titration/ui_state/demo_mode/demo_temp_probe.py

```diff
@@ -54,37 +46,47 @@ def loop(self):
         The function to loop through and display to the LCD screen until a new keypad input
         """
         if self.substate == 1:
-            self.titrator.lcd.print("1: Get Temperature", line=1)
```

Comment:
What about get probe 1 and 2? Then on the probe screen show both temp and resistance.

Reply:
Good idea! I will implement this.